raiden/tests/unit/test_matrix_presence.py

```diff
@@ -133,13 +138,17 @@ def _callback(address, reachability):
 
 @pytest.fixture
 def user_addr_mgr(dummy_matrix_client, address_reachability_callback, user_presence_callback):
```

Comment:
This use of abbreviations is quite uncommon for the codebase

Reply:
True but `user_address_manager` is quite a mouthful. I'll change it if you prefer, though.

Reply:
No strong feelings, just a thought that came to my mind.